src/engine.py

```
@@ -4130,6 +4132,231 @@ async def _handle_transfer_answered(self, channel_id: str, args: list):
+        try:
+            start_task = self._provider_start_tasks.pop(call_id, None)
+            if start_task:
+                start_task.cancel()
+            task = getattr(self, "_pipeline_tasks", {}).pop(call_id, None)
+            if task and not task.done():
+                task.cancel()
+            getattr(self, "_pipeline_queues", {}).pop(call_id, None)
+            getattr(self, "_pipeline_transcript_queues", {}).pop(call_id, None)
+            self._pipeline_forced.pop(call_id, None)
+        except Exception:
+            pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)